ols/src/cache/redis_cache.py

```diff
@@ -59,7 +59,9 @@ def get(self, user_id: str, conversation_id: str) -> Optional[str]:
         """
         key = super().construct_key(user_id, conversation_id)
 
-        return self.redis_client.get(key)
+        value = self.redis_client.get(key)
+        # GET operation might return Awaitable value, so it is time to "realize" it
+        return str(value) if value else None
```

Comment:
https://github.com/openshift/lightspeed-service/pull/351/files#diff-b0c91580ce72dc2568da418b71f17fa43ec39264cfbef4e92cef679a00988eafR53 is changing the datatype, so let's fix this in that PR

Reply:
well that PR got closed so i'm not sure what the plan is here.  @vishnuchalla @xrajesh please decide what you want to do w/ this in light of other refactoring work you're doing around the redis client logic.